terraso_backend/apps/core/management/README.md

```diff
@@ -0,0 +1,34 @@
+# BACKUP / RESTORE
+
+The Terraso backend optionally provides the ability to
+
+- upload a Django object dump to an S3 bucket from an instance (the "Source" instance)
+- download the object dump and restore the data to the database (the "Target" instance)
+- trigger a wipe of the Target instance's DB from the Django admin panel, and restore the data from the Source instance
+
+The last option is only supported if the Source instance is deployed as a Render Web Service, as the Render HTTP API is used to trigger the backup upload.
+
+## Setup
+
+In order to activate the backup/restore functionality, a few environment variables must be set
+
+### Source Instance
+
+- `DB_BACKUP_S3_BUCKET` must point to a S3 bucket where the backups will be stored
+- The `AWS_*` credentials must be valid
+
+
+### Target Instance
+
+- The `ALLOW_RESTORE_FROM_BACKUP` should be set to `true`. This will make the button to trigger the job visible in the Django admin panel.
+- The `DB_RESTORE_CONFIG_FILE` should point to the local of the restore configuration file.
```

Comment:
Question: Why is the DB_RESTORE_CONFIG_FILE value configurable? In what situations will this be changed?

Reply:
My reasoning was
- Don't want to hard code URLs into the code
- Don't want to add like 5 new env variables to the config file just for the backup, which will only be used on the testing environment

It is pretty easy to deploy it and is just a bit cleaner in my opinion.